packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -427,3 +428,35 @@ export const getLinksSupplier = (
     }
   });
 };
+
+/**
+ * Return a copy of the DataFrame with raw data
+ */
+export function applyRawFieldOverrides(data: DataFrame[]): DataFrame[] {
+  if (!data || data.length === 0) {
+    return [];
+  }
+
+  const newData = [...data];
+  const processor = getRawDisplayProcessor();
+
+  for (let frameIndex = 0; frameIndex < newData.length; frameIndex++) {
+    const newFrame = { ...newData[frameIndex] };
+    const newFields = [...newFrame.fields];
+
+    for (let fieldIndex = 0; fieldIndex < newFields.length; fieldIndex++) {
+      newFields[fieldIndex] = {
+        ...newFields[fieldIndex],
+        display: processor,
+        values: new FormattedVector(newFields[fieldIndex].values, processor),
```

Comment:
This should not be needed right? one display processor on the field should be enough

Reply:
I'll look into it

Reply:
Yes, that works. I copy/pasted the old code. Will be awesome to have more documentation about best practices on how to create/update/copy/delete data frames. (ping @marcusolsson)